lib/vehicle.rb

```diff
@@ -11,4 +11,22 @@ def initialize(year, make, model)
       @speeding = false
       @passengers = []
     end
+
+    def speeding?
+        @speeding
+    end
+
+    def speed
+        @speeding = true
+    end
+
+    def add_passenger(passenger)
+        @passengers << passenger
+    end
+
+    def num_adults
+        @passengers.count do |passenger|
```

Comment:
I think count works great! Is there a find_all or filter type method that could work too? LGTM either way.

Reply:
Added some code in my PR that might help here too!

Reply:
notes from discussion:
* we could use passenger#adult? method to determine age